torch/_dynamo/eval_frame.py

```diff
@@ -827,6 +827,24 @@ def result_capturing_wrapper(*graph_inputs):
     assert out_guards is not None, "Failed to produce guards during tracing"
     assert fake_mode is not None
 
+    if tracing_mode == "symbolic":
+
+        def get_dim_constraints(gm):
+            for node in gm.graph.nodes:
+                if "example_value" in node.meta:
+                    return node.meta[
+                        "example_value"
+                    ].fake_mode.shape_env.dim_constraints
```

Comment:
Why not just do it here? (https://github.com/pytorch/pytorch/blob/ae0eb2342d1eab91f7c9ca3125c489756836e6f3/torch/_dynamo/eval_frame.py#L789)